.circleci/config.yml

```diff
@@ -57,23 +60,30 @@ jobs:
     executor: gradle_docker
     steps:
       - setup_build_environment
+      - setup_remote_docker: &latest_remote_docker
+          version: 19.03.12
       - populate_and_save_cache
+      - docker_login
       - gradle:
           args: build dockerBuildImages
       - gradle:
           args: jacocoTestReport
       - codecov/upload:
           flags: unit
-  publish:
+  merge-publish:
     executor: gradle_docker
     steps:
       - setup_build_environment
+      - setup_remote_docker: *latest_remote_docker
+      - docker_login
       - gradle:
-          args: :tag -Prelease
-      - add_ssh_keys:
-          fingerprints:
-            - '9e:5e:1b:28:cf:14:d6:7d:cd:48:64:72:d0:6f:01:c8'
-      - run: git push origin $(./gradlew -q :printVersion)
+          args: dockerPushImages
+  release-publish:
+    executor: gradle_docker
+    steps:
+      - setup_build_environment
```

Comment:
Now, I see two different things - one with docker creds, and one without docker creds (https://github.com/hypertrace/hypertrace-ingester/pull/51). 
Not a blocker for this PR, but we need to see how we want to proceed on forked branch PR.

Reply:
Let's take care of this in separate PR. I somehow missed this.

Reply:
Commented on that PR - please don't propagate removing credentials until we come to an agreed upon approach.